.devcontainer/arch-linux/devcontainer.json

```diff
@@ -19,7 +19,13 @@
             "source": "/tmp/.X11-unix",
             "target": "/tmp/.X11-unix",
             "type": "bind"
+        },
+        {
+            "source": "artery-arch-linux-persist-conan",
+            "target": "/home/devcontainer/.conan2",
+            "type": "volume"
```

Comment:
Can we omit these volumes by using the "use local cache" feature? For example, we could use a *.conan2* directory in the workspace folder, which is already bind-mounted anyway and already has the correct ownership settings.